frontend/apps/app/app/api/chat/route.ts

```diff
@@ -1,121 +1,12 @@
+import { convertSchemaToText } from '@/app/lib/schema/convertSchemaToText'
+import { isSchemaUpdated } from '@/app/lib/vectorstore/supabaseVectorStore'
+import { syncSchemaVectorStore } from '@/app/lib/vectorstore/syncSchemaVectorStore'
 import { mastra } from '@/lib/mastra'
-import type { Schema, TableGroup } from '@liam-hq/db-structure'
 import * as Sentry from '@sentry/nextjs'
 import { NextResponse } from 'next/server'
 
-// Export TableGroupData type for compatibility
-export type TableGroupData = TableGroup
-
-// Convert table data to text document
-const tableToDocument = (
```

Comment:
The process of converting schema information to text has been moved to the following.
frontend/apps/app/app/lib/schema/convertSchemaToText.ts